### PR TITLE
Making new keep notifications look like messages

### PR DESCRIPTION
**Pull request by andrewconner**
@cvializ

### CODE REVIEW

File: packrat/scripts/notices.js [hunk 1]
```diff
@@ -201,76 +202,118 @@ k.panes.notices = k.panes.notices || function () {
...
+      notice.authorShortName = notice.author.firstName;
+    }
+    notice.picturedParticipants.map(formatParticipant);
+    notice.bodyHtmlTree = formatMessage.snippet()(notice.text);
+    return k.render('html/keeper/notice_message', notice, partials);
+  }
+
+  function renderTriggered(notice) {
+    if (notice.fullCategory === 'new_keep') {
+      if (!notice.extra.uriSummary) {  // Backwards compatibility, can be removed once all include the right fields.
```
Comment (andrewconner) [inline packrat/scripts/notices.js]: This can probably be removed now, but I need to test thoroughly.

File: packrat/scripts/notices.js [hunk 2]
```diff
@@ -201,76 +202,118 @@ k.panes.notices = k.panes.notices || function () {
...
+      return renderTriggered(notice);
+    case 'global':
+      return renderGlobal(notice);
+    default:
+      log('#a00', '[renderOne] unrecognized category', notice.category);
+      return '';
+    }
+  }
+
+  function renderMessage(notice) {
```
Comment (andrewconner) [inline packrat/scripts/notices.js:216]: @camhashemi Would you double check this method? All I did was move it from the `switch` above to be in a function (it was getting quite long). I didn't realize you were also in here so we had merge conflicts. Should be correct.

File: packrat/html/keeper/notice_new_keep.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<div class="kifi-notice kifi-notice-message kifi-notice-lines-1{{#isVisited}} kifi-notice-visited{{/isVisited}}" data-id="{{id}}" data-thread="{{thread}}" data-locator="{{locator}}" data-uri="{{url}}" data-created-at="{{time}}" data-category="triggered">
+  <div class="kifi-notice-state"></div>
+  <div class="kifi-notice-mosaic kifi-notice-mosaic-1">
+    <div class="kifi-notice-pic kifi-notice-pic-0 kifi-notice-pic-user" style="background-image:url({{image}})"></div>
+  </div>
+  {{#extra.uriSummary.imageUrl}}
+  <div class="kifi-notice-site-img" style="background-image:url('{{extra.uriSummary.imageUrl}}')"></div>
```
Comment (cvializ) [inline packrat/html/keeper/notice_new_keep.html]: micro-optimization: you don't need the quotes in `url('...')`, like on line 4

File: packrat/scripts/notices.js [hunk 4]
```diff
@@ -201,76 +202,104 @@ k.panes.notices = k.panes.notices || function () {
 
     switch (notice.category) {
     case 'message':
```
Comment (cvializ) [inline packrat/scripts/notices.js:204]: nice refactor :sunglasses:

**Comment by cvializ** [comment]
late but :+1:

File: packrat/scripts/notices.js (hunk 2 above)
Comment (camhashemi) [inline packrat/scripts/notices.js:216]: looks good! thanks for cleaning up!

**Pull request merged**